### ISSUE TITLE
How to debug a migration?

### ISSUE THREAD

**Issue by vasco3** [get-convex/migrations#4]
I ran a migration from the CLI and I get an error

Command to run in prod. Dry run worked well.
`pnpx convex run migrations:run '{"dryRun": false, "fn": "migrations:quoteItem_customglasses_reference_to_id"}' --prod`

Error:
```
migrations:run
failure
514ms
Uncaught Error: Uncaught TypeError: Cannot read properties of undefined (reading 'map')
    at getMigrationState (../../../node_modules/.pnpm/@convex-dev+migrations@0.1.1_@clerk+clerk-react@5.9.1_react-dom@18.3.1_react@18.3.1__react@18_ivxcewucikz3aycsepvjpqtq5e/node_modules/@convex-dev/migrations/src/component/public.ts:220:67)
```

Seems like there is a data inconsistency. It would be great if I could find out the document ID to be able to fix it.

**Comment by vasco3** [comment]
It works in dev, or in prod with dryRun. But in prod dryrun == false it throws the map error.

Actually I just tried the same code in **convex-helpers** and it worked without a problem.

**Comment by ianmacartney** [comment]
I fixed a bug in `@convex-dev/migrations@0.1.3` - sorry about that! You can see the table in the dashboard if you want to see the internals. in this case there was a chance that `next` was undefined as a parameter to `runMigration` and we were trying to map over the non-existent array. I verified the fix and added some more examples to `examples.ts`

**Comment by vasco3** [comment]
thanks!
